.github/workflows/tests.yml

```diff
@@ -411,6 +411,7 @@ jobs:
       artifact-name: image-test
       repository-name: returntocorp/semgrep
   trigger-semgrep-comparison-argo:
+    if: github.event_name == 'pull_request'
```

Comment:
I think we probably also want to run this when we merge to develop? cc @IagoAbal

Reply:
Eventually we could perhaps run a "full" comparison with each merge to `develop`. But PRs only is good for now IMO.